scripts/EN/Harry

```diff
@@ -7329,26 +7329,26 @@ Area(0x107); // 0x94b4: 00001026
 CleanScreen(); // 0x94ba: 00000064
 LoadBackground(0xab); // 0x94be: 00001069
 DrawBackground(); // 0x94c4: 00000065
-ShowText((3007)"After looking over the manuscript again I let out a long sign, and neatly pushed the stack of papers together and placed them in the envelope I had prepared."); // 0x94c8: 0000100c
+ShowText((3007)"After looking over the manuscript again I let out a long sigh, W1and neatly pushed the stack of papers together and placed them in the envelope I had prepared."); // 0x94c8: 0000100c
 HandleInput(); // 0x94ce: 00000010
-ShowText((3008)"I licked the seal of the envelope and then tightly sealed it. The act of properly sealing the envelope with such feeling seemed like a certain kind of ceremony."); // 0x94d2: 0000100c
+ShowText((3008)"I licked the seal and then tied a string around it to tightly seal the envelope.¥nW2The act of properly sealing the envelope with such feeling seemed like a certain kind of ceremony."); // 0x94d2: 0000100c
 HandleInput(); // 0x94d8: 00000010
 PlaySecondSoundFx(0x1f); // 0x94dc: 00001073
 CleanScreen(); // 0x94e2: 00000064
 LoadBackground(0xac); // 0x94e6: 00001069
 DrawBackground(); // 0x94ec: 00000065
-ShowText((3009)"¥"Daddy¥" I turn around and Cheryl is standing there pouting. ¥"Ah, sorry. I have just finished. Sorry to have kept you waiting.¥" Cheryl smiled, returned to her room downstairs, and pulled several of her best outfits out of a wardrobe."); // 0x94f0: 0000100c
+ShowText((3009)"¥"Daddy¥"¥nW2I turn around and Cheryl is standing there pouting.¥nW2¥"Ah, W2sorry. W2I have just finished. W2Sorry to have kept you waiting.¥"¥n¥nCheryl smiled, W1returned to her room downstairs, W1and pulled several of her best outfits out of a wardrobe."); // 0x94f0: 0000100c
 HandleInput(); // 0x94f6: 00000010
 CleanScreen(); // 0x94fa: 00000064
 LoadBackground(0x0); // 0x94fe: 00001069
 DrawBackground(); // 0x9504: 00000065
-ShowText((3010)"Tomorrow is the deadline. If I mail it tonight by express mail it should arrive at the proofreader in time. ¥"I'll just be a little longer.¥" I said that to her in front of her room and then placed my hand on the front door."); // 0x9508: 0000100c
+ShowText((3010)"Tomorrow is the deadline.¥nW2If I mail it tonight by express mail it should arrive at the proofreader in time.¥n¥nW2¥"I'll just be a little longer.¥"¥n¥nW2I said that to her in front of her room W1and then placed my hand on the front door."); // 0x9508: 0000100c
 HandleInput(); // 0x950e: 00000010
 CleanScreen(); // 0x9512: 00000064
 LoadBackground(0xad); // 0x9516: 00001069
 DrawBackground(); // 0x951c: 00000065
 PlaySoundFx(0x17); // 0x9520: 00001072
-ShowText((3011)"Ziriririri... The phone rings. Who is it?"); // 0x9526: 0000100c
+ShowText((3011)"Ri-i-ingg...¥nW2The phone rings.¥nW2Who is it?"); // 0x9526: 0000100c
```

Comment:
Added English ringing sound for phones. Made it the same as the one in the school for continuity.